client/dashboard/me/security-two-step-auth/common/print-backup-codes.tsx

```diff
@@ -147,13 +147,7 @@ export default function PrintBackupCodes( {
 		<Card>
 			<CardBody>
 				<VStack spacing={ 4 }>
-					<SectionHeader
-						title={ __( 'Print or download your backup codes' ) }
-						level={ 3 }
-						description={ __(
```

Comment:
Removing as it's duplicated in the existing screen.

<img width="786" height="773" alt="Image" src="https://github.com/user-attachments/assets/2bdacdd0-a65e-4d0b-a9a1-e9f8597d9155" />